internal/token/token_test.go

```diff
@@ -0,0 +1,143 @@
+package token_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/authd-oidc-brokers/internal/fileutils"
+	"github.com/ubuntu/authd-oidc-brokers/internal/providers/info"
+	"github.com/ubuntu/authd-oidc-brokers/internal/token"
+	"golang.org/x/oauth2"
+)
+
+var testToken = token.AuthCachedInfo{
+	Token: &oauth2.Token{
+		AccessToken:  "accesstoken",
+		RefreshToken: "refreshtoken",
+	},
+	RawIDToken: "rawidtoken",
+	UserInfo: info.User{
+		Name:  "foo",
+		UUID:  "saved-user-id",
+		Home:  "/home/foo",
+		Gecos: "foo",
+		Shell: "/usr/bin/bash",
+		Groups: []info.Group{
+			{Name: "token-test-group", UGID: "12345"},
+		},
+	},
+}
+
+func TestCacheAuthInfo(t *testing.T) {
+	t.Parallel()
+
+	tests := map[string]struct {
+		setup       func(t *testing.T, path string)
+		expectError bool
```

Comment:
s/expect/want

Let's keep the wording the same as we have for all of our projects.

Reply:
done

Reply:
Same thing for "fixed for one test, but others still have the same behavior"

Reply:
Should now all be updated